sky/packages/sky/lib/src/widgets/editable_text.dart

```diff
@@ -128,6 +128,8 @@ class EditableString implements KeyboardClient {
     selection = new TextRange(start: start, end: end);
     onUpdated();
   }
+
+  void submit(SubmitAction action) {}
```

Comment:
yes - the next change on the Sky side will add an API for handling the submit event